client/src/commands/run.ts

```diff
@@ -49,8 +49,7 @@ async function getSelectedRegions(
       "sas/getFoldingBlock",
       {
         textDocument: { uri: window.activeTextEditor.document.uri.toString() },
-        line,
-        col,
+        position: { line, col },
```

Comment:
I changed this because all the onXXX() events at server side have a structure like param{position{line: xx, col: xx}}, so I encapsulated a common function "dispatch" which resolves that structure and handle common logics. Changing this is just to make it the same like all the other interfaces, and adapt the dispatch() function. Considering this is a sas only call, this shouldn't have other side effects that I can know.

It's a bug at the line 228-229, I'll fix it. For any other concerns pls let me know.